### PR TITLE
add new parser xray

### PR DESCRIPTION
**Pull request by renejal**
From our organization we want to contribute with a new xray parser for https://jfrog.com/help/r/jfrog-cli/on-demand-binary-scan

image

### CODE REVIEW

**Comment by kiblik** [comment]
I'm happy to see this parser. We bumped into the same missing piece so we wanted to propose our version of parser. But your is already covering more use cases.

I will add some of our reports + unit tests.

But, I'm not 100% sure about the parser's name. I fully understand that it is coming from the official name of the method, how to obtain the mentioned report. Just not sure.

**Comment by renejal** [comment]
> I'm happy to see this parser. We bumped into the same missing piece so we wanted to propose our version of parser. But your is already covering more use cases.
> 
> I will add some of our reports + unit tests.
> 
> But, I'm not 100% sure about the parser's name. I fully understand that it is coming from the official name of the method, how to obtain the mentioned report. Just not sure.

File: .dryrunsecurity.yaml
Comment (mtesauro) [inline .dryrunsecurity.yaml:1]: This file isn't part of the parser so please remove from this PR

File: docs/content/en/getting_started/upgrading.md
Comment (mtesauro) [inline docs/content/en/getting_started/upgrading.md:1]: Same for this file. 

Once these extra files are removed, I'm ready to approve this PR.

**Review by mtesauro — COMMENTED** [review]
Clean up 2 extra files in this PR and it will be good to go

**Pull request closed without merging**